Utilities/DataLoading/EdFi.SmokeTest.Console/Program.cs

```diff
@@ -6,22 +6,22 @@
 using System;
 using System.Diagnostics;
 using System.Net;
+using System.Threading.Tasks;
+using Autofac;
 using EdFi.LoadTools;
 using EdFi.LoadTools.Engine;
 using EdFi.LoadTools.SmokeTest;
 using EdFi.SmokeTest.Console.Application;
 using log4net;
 using log4net.Config;
-using SimpleInjector;
 
 namespace EdFi.SmokeTest.Console
 {
     public partial class Program
```

Comment:
You've ripped out the partial functionality, so the class can just be static now.

Reply:
done